### PR TITLE
etching example + cleanup

### PR DESCRIPTION
**Pull request by m-aboelenein**
- Adds an etching example to the test app
- Export Wallet instance instead of static class
- update sats-connect-core to the latest version (bug-fix - btc_providers array mutated on connect)

### CODE REVIEW

File: example/src/components/EtchRunes/index.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    [addresses]
+  );
+
+  const paymentAddress = useMemo(
+    () => addresses.find((a) => a.purpose === AddressPurpose.Payment)?.address || '',
+    [addresses]
+  );
+
+  const onClickEstimate = async () => {
+    const response = await Wallet.request('runes_estimateEtch', {
```
Comment (emilcondrea) [inline example/src/components/EtchRunes/index.tsx:32]: We should add an amount & cap. Open mints without a cap are unmintable.

Comment (m-aboelenein) [reply example/src/components/EtchRunes/index.tsx:32]: what happens on the service if you submit an order with `isMintable=true` and no amount & cap?

Comment (emilcondrea) [reply example/src/components/EtchRunes/index.tsx:30]: we are working on adding all validations of required parameters & valid combinations, we are running short on time on adding API unit tests. This uncapped openmint that creates unmintable runes is not even documented in runes spec, I found out about it from ord code.

The desired behavior is that API should reject creating unmintable runes, we need to add all scenarios this week.

Comment (victorkirov) [reply example/src/components/EtchRunes/index.tsx:32]: It will throw a 400

Comment (victorkirov) [reply example/src/components/EtchRunes/index.tsx:32]: That would be caught in the sats-connect method and would return an error result here, so maybe that's enough?

**Review by victorkirov — APPROVED** [review]

**Pull request merged**